include/radix/core/gl/OpenGL.hpp

```diff
@@ -1,6 +1,10 @@
 #ifndef RADIX_CORE_GL_OPENGL_HPP
 #define RADIX_CORE_GL_OPENGL_HPP
 
+#ifdef _WIN32
+#define RADIX_GL_USE_GLAD
```

Comment:
I'm not forcing GLAD on windows, I'm forcing GLAD in visual studio. Since CMakeLists isn't relevant there.